Update semantic search tutorial to include instructions on using the … (#3261)
elastic/docs-content: solutions/search/semantic-search/semantic-search-semantic-text.md

1. The name of the field to contain the generated embeddings.
2. The field to contain the embeddings is a `semantic_text` field. Since no `inference_id` is provided, the default endpoint `.elser-2-elasticsearch` for the `elasticsearch` service is used. To use a different {{infer}} service, you must create an {{infer}} endpoint first using the [Create {{infer}} API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-inference-put) and then specify it in the `semantic_text` field mapping using the `inference_id` parameter.

> 3. To try the ELSER model on the Elastic Inference Service explicitly set the `inference_id` to `.elser-2-elastic`, see [Using `semantic_text` with ELSER on EIS](https://www.elastic.co/docs/reference/elasticsearch/mapping-reference/semantic-text#using-elser-on-eis) to learn how to set the `inference_id`. 

Reviewer: ```suggestion
3. To try the ELSER model on the Elastic Inference Service, explicitly set the `inference_id` to `.elser-2-elastic`. For instructions, refer to [Using `semantic_text` with ELSER on EIS](https://www.elastic.co/docs/reference/elasticsearch/mapping-reference/semantic-text#using-elser-on-eis). 
```

Reviewer 2: IMO this shouldn't be included in the numbered list, since that's intended for the [code callouts](https://docs-v3-preview.elastic.dev/elastic/docs-builder/tree/main/syntax/code#code-callouts) and there isn't a third callout in the block. I think it's sufficient to have it as a sentence that follows the bulleted list.
